code/src/renderer/components/CommandIcon.js

```diff
@@ -17,7 +17,7 @@ export default function CommandIcon(props) {
 function computeIcon(name) {
   return {
     Run: { color: "green", className: "fas fa-play" },
-    Test: { color: "red", className: "fas fa-graduation-cap" },
+    Test: { color: "red", className: "fas fa-vial" },
```

Comment:
The play button is also green so I guess green is taken. Maybe a purple vial? Red is scary. Purple is alluring. 

<img width="190" alt="Screen Shot 2021-02-03 at 6 05 15 PM" src="https://user-images.githubusercontent.com/297042/106834471-626f3380-664a-11eb-8608-cc875c5248ff.png">

Reply:
We have a green play icon for "run", so I didn't want to reuse that color. The localhost scheme editor has a button reading "TEST", I think.

Reply:
Accessibility nits: It'd be great to add tooltips (even via the title= attribute) to these buttons at some point. Also to make them actual <button> instead of <span>s, in case one day we have a screen reader using student. https://karlgroves.com/2013/05/14/links-are-not-buttons-neither-are-divs-and-spans